### DISCUSSION TITLE
K8s 1.24 - Formalize docs release strategy with docs localization owners

### DISCUSSION THREAD

**Discussion by nate-double-u** [kubernetes/sig-release#1821] [General]
Hello 👋, I'm Nate Waddington, the SIG Release Docs lead for the 1.24 release. I'm opening this discussion to coordinate the docs release (scheduled tentnatively for April 19, 2022 with localization owners. I look forward to working with everyone! Let me know if you have any questions! Thank you! 🙏

cc: @kubernetes/sig-docs-fr-owners @kubernetes/sig-docs-de-owners @kubernetes/sig-docs-hi-owners @kubernetes/sig-docs-id-owners @kubernetes/sig-docs-ja-owners @kubernetes/sig-docs-ru-owners @kubernetes/sig-docs-pl-owners @kubernetes/sig-docs-ko-owners @kubernetes/sig-docs-pt-owners @kubernetes/sig-docs-es-owners @kubernetes/sig-docs-zh-owners @kubernetes/sig-docs-vi-owners @kubernetes/sig-docs-uk-owners @kubernetes/sig-docs-it-owners

**Comment by raelga** [comment]
I'm in for the kubernetes/sig-docs-es team

**Comment by Fale** [comment]
Count me in for the kubernetes/sig-docs-it team!

**Comment by MaxymVlasov** [comment]
As always, I'm in for kubernetes/sig-docs-uk-owners

**Comment by mfilocha** [comment]
Do we need to report active owners? I remember it was requested few releases back, but I don't see this requirement any longer.

I assume this is voluntary now, @nate-double-u?

**Reply by nate-double-u** [reply]
I'm actually not sure, I haven't heard of this requirement. Where was it written?

**Reply by mfilocha** [reply]
I think that was a kind of recommendation coming from one of previous Release Docs lead. But I don't think it should be mandatory for every release.

**Comment by tengqm** [comment]
ping me if need help on reference docs generation or zh localization.

**Comment by nasa9084** [comment]
ping me if need help on ja localization

**Comment by anubha-v-ardhan** [comment]
I'm in for kubernetes/sig-docs-hi team. 
We are the newest localization team and localization is yet to go live on k8s.io(we expect to launch soon as we are almost done with [minimum required content](https://kubernetes.io/docs/contribute/localization/#minimum-required-content))

**Comment by nate-double-u** [comment]
Hello localization team leads! I don't think any action is required from you, but I wanted to let you know that we are on track for the release on May 3, 2022. The k/website repo will be frozen on May 2, 2022 until the release is finished. 

Let me know if I can help with anything! Thanks!
